evap/student/templates/student_vote.html

```diff
@@ -273,16 +287,19 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
                     presaveButton.removeClass("disabled");
                     presaveButton.text(originalPresaveText);
                 }, 2500);
-            }
+            });
 
             // handle text_results_publish_confirmation checkbox changes
-            function textResultsPublishConfirmationTopChanged() {
-                var topIsChecked = $("#text_results_publish_confirmation_top").prop('checked');
-                $("#text_results_publish_confirmation_bottom").prop('checked', topIsChecked);
-                $("#bottom_text_results_publish_confirmation_card").toggle(!topIsChecked);
-            }
-            function textResultsPublishConfirmationBottomChanged() {
-                $("#text_results_publish_confirmation_top").prop('checked', $("#text_results_publish_confirmation_bottom").prop('checked'));
+            if (textResultsPublishConfirmation.bottomCard) {
+                textResultsPublishConfirmation.top.addEventListener("change", () => {
+                    const isChecked = textResultsPublishConfirmation.top.checked;
+                    textResultsPublishConfirmation.bottom.checked = isChecked;
+                    textResultsPublishConfirmation.bottomCard.classList.toggle("d-none", isChecked);
+                });
+                textResultsPublishConfirmation.bottom.addEventListener("change", () => {
+                    textResultsPublishConfirmation.top.checked = textResultsPublishConfirmation.bottom.checked;
+                    sisyphus.saveAllData();
```

Comment:
please add a comment why this is needed

Reply:
I came up with this:
```
// The top checkbox should only be visually checked without triggering the change event,
// which would hide the bottom card.
// To keep the top checkbox checked (after a reload or submit), save the form manually.
```
Too long? Too verbose? Then I’d like another proposal (`my` :gb: `is` :exclamation: :yellow_square: `from` :egg: )

Reply:
it's fine 👍